sonar-project.properties

```diff
@@ -0,0 +1,18 @@
+# SonarCloud project key
+sonar.projectKey=RGPosadas_Mull
+
+# Source code encoding
+sonar.sourceEncoding=UTF-8
+
+#Metadata
+sonar.links.homepage=https://github.com/RGPosadas/Mull
+sonar.links.ci=https://travis-ci.com/github/RGPosadas/Mull
+sonar.links.scm=https://github.com/RGPosadas/Mull
+
+# SonarQube properties
```

Comment:
Are SonarQube and SonarCloud the same thing?

Reply:
No they are not, I forgot to change the name since I copied pasted this from WayFinder. I shall correct it now. Thanks for catching it!